.github/workflows/test-contracts.yml

```
@@ -82,6 +82,12 @@ jobs:
 
       - name: Slither static analyser
         uses: crytic/slither-action@v0.3.0
-        continue-on-error: true
+        id: slither
         with:
-          node-version: ${{ matrix.node_version }}
+          fail-on: config
```

Per the PR description and the setup I normally use [here](https://github.com/ScopeLift/foundry-template/blob/main/.github/workflows/ci.yml), I think this is set wrong?
```suggestion
          fail-on: none
```

---

It's set in the `slither.config.json` file.

---

Ah I see, thanks!